app/src/main/java/edu/mills/cs115/AZCategoryFragment.java

```diff
@@ -62,25 +70,28 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             toast = Toast.makeText(this.getContext(), "Database Unavailable", Toast.LENGTH_LONG);
             toast.show();
         }
+    }
 
-
-        CategoryAdapter categoryAdapter = new CategoryAdapter(categories);
-        recyclerView.setAdapter(categoryAdapter);
+    /**
+     * Creates the RecyclerView adapter for displaying category CardView's.
```

Comment:
Creates the RecyclerView adapter for displaying category CardView's[CardViews?].